app/helpers/application_helper.rb

```diff
@@ -8,6 +8,25 @@ def boolean_red_green_tag(bool, text = nil)
     content_tag(:strong, text, class: "govuk-tag govuk-tag--#{colour}")
   end
 
+  def boolean_submitted_passed_tag(status, text = nil)
+    case status.downcase
```

Comment:
Cosmetic:
```
color = case status.downcase
when "foo"
  "bar"
when "baz"
  "foobar"
end
```
Or
```
{ 
  "foo" => "bar",
  ...
}[status]
````

Reply:
`#fetch`? 😅

Reply:
Also as we're already using govuk-components it might make sense to use `govuk_tag` here, it supports [all the colours](https://govuk-components.netlify.app/components/tag/#coloured-tags).